src/modules/filter-sort/components/filter-dropdowns/filter-dropdowns.jsx

```diff
@@ -111,22 +108,10 @@ export default class FilterSearch extends Component {
     this.changeMaxFees = this.changeMaxFees.bind(this);
     this.changeMaxSpread = this.changeMaxSpread.bind(this);
     this.goToPageOne = this.goToPageOne.bind(this);
-    this.changeHidePastCutoff = this.changeHidePastCutoff.bind(this);
     this.changeExperimentalInvalid = this.changeExperimentalInvalid.bind(this);
     this.changeHideInsureMarkets = this.changeHideInsureMarkets.bind(this);
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (
-      nextProps.hasPositionsInCutoffMarkets !==
```

Comment:
Oh I bet the new filter won't show markets you have positions in if they're past they don't have enough IR stake.

This but reminded me of that logic.

Reply:
Yea, we just want to show the pop up modal. I would think.  I do not know what the model says but might as will pop up